src/docfx/template/TemplateSchema.cs

```diff
@@ -28,15 +33,18 @@ public TemplateSchema(string schemaName, string schemaDir, string contentTemplat
 
             SchemaName = SchemaName;
             IsData = GetIsDataCore(schemaName, contentTemplateDir);
-            TransformMetadata = File.Exists(_metadataTransformJsPath = Path.Combine(contentTemplateDir, $"{schemaName}.mta.json.js"));
+            TransformMetadata = File.Exists(MetadataTransformJsPath = Path.Combine(contentTemplateDir, $"{schemaName}.mta.json.js"));
```

Comment:
`MetadataTransformJsPath` set but not used?

Reply:
Not yet

Reply:
is it going to be used in this PR or?

Reply:
removed